core/src/test/java/org/apache/iceberg/TestRemoveSnapshots.java

```diff
@@ -1102,19 +1050,17 @@ public void testExpireWithDefaultSnapshotAge() {
     // rely solely on default configs
     removeSnapshots(table).deleteWith(deletedFiles::add).commit();
 
-    Assert.assertEquals(
-        "Should not change current snapshot", thirdSnapshot, table.currentSnapshot());
-    Assert.assertEquals("Should keep 1 snapshot", 1, Iterables.size(table.snapshots()));
-    Assert.assertEquals(
-        "Should remove expired manifest lists",
-        Sets.newHashSet(
-            firstSnapshot.manifestListLocation(), secondSnapshot.manifestListLocation()),
-        deletedFiles);
+    assertThat(table.currentSnapshot()).isEqualTo(thirdSnapshot);
+    assertThat(table.snapshots()).hasSize(1);
+    assertThat(deletedFiles)
+        .isEqualTo(
```

Comment:
`.containsExactly()`

Reply:
The order of `firstSnapshot.manifestLocation` and `secondSnapshot.manifestLocation` depends on the hashset values, so the order is not always the same. So, this part needs to use `isEqualTo`, or `containsExactlyInAnyOrder`.

Reply:
let's use `containsExactlyInAnyOrder` then

Reply:
Okay, thank you!